### DISCUSSION TITLE
How I stop smooth-scrollbar?

### DISCUSSION THREAD

**Discussion by Minozzzi** [idiotWu/smooth-scrollbar#521] [Q&A]
Hello guys, how are you?

I need help to stop the smooth-scroll when I get to a certain container on my page, how could I do that?

**Comment by sadeghbarati** [comment]
Hi, do you mean to **disable** and **enable** `smooth-scrollbar`?

Check this comment https://github.com/idiotWu/smooth-scrollbar/discussions/361#discussioncomment-854079, this is about Modal case but It should work in your issue too,
You can also rename "ModalPlugin" to anything more meaningful about your case

**Reply by Minozzzi** [reply]
Thanks for the answer, but it's not what I need. I would like it to go back to native scrolling and not stop scrolling completely
